templates/powershell/build/Extract-CodeVersion.ps1

```diff
@@ -103,15 +103,14 @@ try {
     if ($IsDefaultBranchBuild -eq "False") {
         #Check if the version is upgraded
         if (([version]$appVersion).CompareTo(([version]$oldAppVersion)) -gt 0) {
-            Write-Output "${functionName}:appVersion upgraded"    
+            Write-Output "${functionName}:Version increment valid '$oldAppVersion' -> '$appVersion'."    
         }
         else {
-            Write-Output "${functionName}:appVersion not upgraded"   
+            Write-Output "${functionName}:Version increment invalid '$oldAppVersion' -> '$appVersion'."   
```

Comment:
Can we add a bit more to the end of it. I.e.. "Please increment the version to run the CI process" or similar

Reply:
Done